control-plane/build-support/functions/10-util.sh

```diff
@@ -695,6 +713,7 @@ function set_changelog {
 	#   $2 - Version
 	#   $3 - Release Date
 	#   $4 - The last git release tag
+  #   $5 - Pre-release version
 	#
```

Comment:
```suggestion
	#   $4 - The last git release tag
	#   $5 - Pre-release version
	#
```

Reply:
it's all fun and games until you're missing `go fmt` for other languages :(